backend/tracim_backend/lib/core/user.py

```diff
@@ -958,6 +972,7 @@ def _check_username_correctness(self, username: str) -> bool:
     def update(
         self,
         user: User,
+        external_id: typing.Optional[str] = None,
```

Comment:
I think you should put this new field at the end of parameters if you want to be sure not to break the interface.

Example: if somewhere in the code stuff is written like `obj.update(user, "new name", "new email")` inserting the external_id breaks the behavior

Reply:
Fixed in last commit